cmd/pj-rehearse/server.go

```diff
@@ -274,6 +281,16 @@ func (s *server) handleIssueComment(l *logrus.Entry, event github.IssueCommentEv
 	}
 }
 
+func (s *server) getCandidate(pullRequest github.PullRequest) (rehearse.RehearsalCandidate, error) {
+	// We can't just use the base SHA from the pullRequest as there may be changes to the branches head in the meantime
+	repo := pullRequest.Base.Repo
+	baseSHA, err := s.ghc.GetRef(repo.Owner.Login, repo.Name, fmt.Sprintf("heads/%s", pullRequest.Base.Ref))
```

Comment:
This method is named oddly. It actually returns a SHA:
https://github.com/kubernetes/test-infra/blob/5022c525793a6f4a741ae47f45e5049b4ec4ad5c/prow/github/client.go#L3359-L3360